src/Header/Component.tsx

```
@@ -3,9 +3,18 @@ import { getCachedGlobal } from '@/utilities/getGlobals'
 import React from 'react'
 
 import type { Header } from '@/payload-types'
+import { getMeUser } from '@/utilities/getMeUser'
+
+export const dynamic = "force-dynamic"
 
 export async function Header() {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Header'. Consider to delete it or rename it.

'Header' is defined here:


(lint/suspicious/noRedeclare)